### ISSUE TITLE
Timeliner Not Loading

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#4775]
# Current Behavior
Existing Timelines are sometimes not loading in avalon-staging. Example: https://avalon-staging.dlib.indiana.edu/timelines/10, timelines created from https://avalon-staging.dlib.indiana.edu/media_objects/t148fh12j. The Timeliner launches when opening the timeline, but the loading spinner spins endlessly in this case.

# Done Criteria
- [ ] Problem causing Timeliner to not load is investigated and addressed if possible (i.e., not due to existing bad data in a Timeliner manifest that causes this)

**Comment by cjcolvar** [comment]
AFAICT this is only occuring for this one video which plays outside of timelines.  @Dananji would you be able to take a look since it is failing in the JS as far as I can tell.

**Comment by cjcolvar** [comment]
@joncameron What do you think about closing this issue?  It feels like an edge case that might take a long time to get to the bottom of.  Maybe it is worth exploring switching out the mediaelement.js player in the timeliner for @Dananji 's video.js based react component or some other iiif player component?

**Comment by joncameron** [comment]
Agreed, I'm going to close this one. It was working in other scenarios so I'm going to call it edge case and just create a new issue if it pops up again.
